src/errors/test-run/templates.js

```diff
@@ -78,6 +78,12 @@ export default {
         ${escapeHtml(err.errMsg)}
     `),
 
+    [TYPE.uncaughtErrorInSnapshotExtensionCode]: err => markup(err, `
+        An error occurred when trying to calculate a custom ${err.property} property:
```

Comment:
Of selector? Don't think so. I have no other thoughts though.
@kirovboris, whose property is this?